projects/packages/forms/src/blocks/contact-form/components/jetpack-field-controls.js

```diff
@@ -163,105 +157,226 @@ const JetpackFieldControls = ( {
-								value={ attributes.buttonBorderWidth }
-								initialPosition={ 1 }
-								onChange={ setNumberAttribute( 'buttonBorderWidth' ) }
-								min={ 0 }
-								max={ 100 }
-								__nextHasNoMarginBottom={ true }
-							/>
-							<RangeControl
-								label={ __( 'Button Border Radius', 'jetpack-forms' ) }
-								value={ attributes.buttonBorderRadius }
-								initialPosition={ 0 }
-								onChange={ setNumberAttribute( 'buttonBorderRadius' ) }
-								min={ 0 }
-								max={ 100 }
-								__nextHasNoMarginBottom={ true }
-							/>
-						</>
-					) }
-					{ ( ! isChoicesBlock || formStyle === FORM_STYLE.OUTLINED ) && (
-						<>
-							<RangeControl
-								label={ __( 'Border Width', 'jetpack-forms' ) }
-								value={ attributes.borderWidth }
-								initialPosition={ 1 }
-								onChange={ setNumberAttribute( 'borderWidth' ) }
-								min={ 0 }
-								max={ 100 }
-								__nextHasNoMarginBottom={ true }
-							/>
-							<RangeControl
-								label={ __( 'Border Radius', 'jetpack-forms' ) }
-								value={ attributes.borderRadius }
-								initialPosition={ 0 }
-								onChange={ setNumberAttribute( 'borderRadius' ) }
-								min={ 0 }
-								max={ 100 }
-								__nextHasNoMarginBottom={ true }
-							/>
-						</>
-					) }
-				</PanelBody>
-				<PanelBody title={ __( 'Label Styles', 'jetpack-forms' ) } initialOpen={ false }>
-					<BaseControl>
+					</ToolsPanelItem>
+				</ToolsPanel>
+				<ToolsPanel
+					label={ __( 'Label typography', 'jetpack-forms' ) }
+					dropdownMenuProps={ toolsPanelDropdownMenuProps }
+				>
+					<ToolsPanelItem
+						hasValue={ () => !! attributes.labelFontSize }
+						label={ __( 'Size', 'jetpack-forms' ) }
+						onDeselect={ () =>
+							setAttributes( {
+								labelFontSize: undefined,
+							} )
+						}
+						isShownByDefault
+					>
 						<FontSizePicker
 							withReset={ true }
 							size="__unstable-large"
 							__nextHasNoMarginBottom
 							onChange={ labelFontSize => setAttributes( { labelFontSize } ) }
 							value={ attributes.labelFontSize }
 						/>
-					</BaseControl>
-					<BaseControl>
+					</ToolsPanelItem>
+					<ToolsPanelItem
+						hasValue={ () => !! attributes.labelLineHeight }
+						label={ __( 'Line height', 'jetpack-forms' ) }
+						onDeselect={ () =>
+							setAttributes( {
+								labelLineHeight: undefined,
+							} )
+						}
+					>
 						<LineHeightControl
 							__unstableInputWidth="100%"
 							__nextHasNoMarginBottom={ true }
 							value={ attributes.labelLineHeight }
 							onChange={ setNumberAttribute( 'labelLineHeight', parseFloat ) }
 							size="__unstable-large"
 						/>
-					</BaseControl>
-				</PanelBody>
+					</ToolsPanelItem>
+				</ToolsPanel>
+				<ToolsPanel
+					label={ __( 'Border', 'jetpack-forms' ) }
+					panelId={ clientId }
+					dropdownMenuProps={ toolsPanelDropdownMenuProps }
+				>
+					{ ( isChoicesBlock || blockStyle === 'button' ) && (
```

Comment:
This replicates what's in trunk, but I'm a bit confused by this condition, because it looks like the 'button' style is only available on 'choices' blocks, so the whole `|| blockStyle === 'button'` is meaningless.

Also, when the block style is something other than button, these button border styles have no effect, but I'm not sure if that's the fault of this condition (perhaps it should be an `&&`?) or if the css is broken.

I'll make sure to log these bugs if they're not already.